babel/util.py

```diff
@@ -220,79 +220,6 @@ def wraptext(text, width=70, initial_indent='', subsequent_indent=''):
     return wrapper.wrap(text)
 
 
-class odict(dict):
```

Comment:
Since the old `odict` is basically `collections.OrderedDict`, we could maybe do something like

```python
# TODO (Babel 3.x): Remove this re-export
odict = collections.OrderedDict
```

in case someone is still using this for some reason.

The other changes are fine, though - no need to use the `odict` alias internally.